.github/pull_request_template.md

```diff
@@ -0,0 +1,40 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
+<!--- Describe your changes in detail -->
+
+## Related Issue
+<!--- This project only accepts pull requests related to open issues -->
```

Comment:
This is potentially too complicated to put in a template, but does this apply whether they're fixing a bug documented in a git issue or a redmine ticket?

Reply:
We should say issue or redmine ticket.

Reply:
@jessemapel :+1:

Reply:
Taking the ticket number out of the title is also okay with me from a watchdog perspective. The git log parsing we used to use to gather the changes from the last week for the redmine notification doesn't work reliability anyway. @SgStapleton Any thoughts?